src/test/java/teammates/test/cases/search/FeedbackResponseCommentSearchTest.java

```diff
@@ -0,0 +1,97 @@
+package teammates.test.cases.search;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.FeedbackResponseCommentSearchResultBundle;
+import teammates.common.datatransfer.attributes.FeedbackResponseCommentAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.storage.api.FeedbackResponseCommentsDb;
+
+/**
+ * SUT: {@link FeedbackResponseCommentsDb},
+ *      {@link teammates.storage.search.FeedbackResponseCommentDocument},
+ *      {@link teammates.storage.search.FeedbackResponseCommentSearchQuery}.
+ */
+public class FeedbackResponseCommentSearchTest extends BaseSearchTest {
+
+    @Test
+    public void allTests() {
+        FeedbackResponseCommentsDb commentsDb = new FeedbackResponseCommentsDb();
+
+        FeedbackResponseCommentAttributes frc1I1Q1S1 = dataBundle.feedbackResponseComments
+                .get("comment1FromT1C1ToR1Q1S1C1");
+        FeedbackResponseCommentAttributes frc1I1Q2S1 = dataBundle.feedbackResponseComments
+                .get("comment1FromT1C1ToR1Q2S1C1");
+        ArrayList<InstructorAttributes> instructors = new ArrayList<InstructorAttributes>();
+        instructors.add(dataBundle.instructors.get("instructor1OfCourse1"));
+
+        ______TS("success: search for comments in data bundle; query string does not match any comment");
```

Comment:
Added test cases where an instructor owns more than one courses.Also, we could have had one more test case to check if the instructor had privileges to view comments, but that check is made in `InstructorSearchPageAction`.